src/main/java/it/ipzs/fedauthority/oidclib/GlobalOptions.java

```diff
@@ -0,0 +1,127 @@
+package it.ipzs.fedauthority.oidclib;
+
+import java.util.Collections;
+import java.util.Set;
+
+import it.ipzs.fedauthority.oidclib.exception.ConfigException;
+import it.ipzs.fedauthority.oidclib.exception.OIDCException;
+import it.ipzs.fedauthority.oidclib.util.ArrayUtil;
+import it.ipzs.fedauthority.oidclib.util.Validator;
+
+
+public class GlobalOptions<T extends GlobalOptions<T>> {
+
+	public static final int DEFAULT_EXPIRING_MINUTES = 30;
+
+	public static final String DEFAULT_SIGNING_ALG = "RS256";
```

Comment:
we should take these parameters from a ini file or from the ENV, not statically compiled in the code.

Reply:
These properties were configured from a previous version of a library shared in the last projects (i.e. eudi-wallet-it-openid-credential-issuer), there will be a refactor about it in a next version.

Reply:
Updated